cdap-security/src/main/java/co/cask/cdap/security/authorization/AuthorizerInstantiator.java

```diff
@@ -107,12 +107,16 @@ public synchronized Authorizer get() {
     }
 
     if (!authorizationEnabled) {
-      LOG.debug("Authorization is disabled. Using a no-op authorizer.");
+      LOG.info("Authorization is disabled. Authorization can be enabled  by setting " +
```

Comment:
Changed the first one back to debug. The second one under `if (!authenticationEnabled) {` should be info because that is an incorrect configuration and user should be told how to fix it.